Makefile

```diff
@@ -108,6 +101,13 @@ coverage: test
 	cat $(COVERAGE_FILE) | grep -v "_mock.go" > $(COVERAGE_FILE).no-mocks
 	go tool cover -func=$(COVERAGE_FILE).no-mocks
 
+ginkgo:
+	mkdir -p $(CURDIR)/bin
+	GOBIN=$(CURDIR)/bin go install github.com/onsi/ginkgo/v2/ginkgo@latest
+
+test-e2e: ginkgo
+	./bin/ginkgo $(GINKGO_ARGS) -v --json-report $(LOG_ARTIFACTS_DIR)/ginkgo.json ./tests/e2e/...
```

Comment:
What is the benefit of using ginkgo as the test runner? Is it only to produce better logs?

Reply:
there is no runtime difference from using go or ginkgo to run the test, yes I am using the ginkgo binary to run the test to get better logs and the report file. So I am not hard on this one, if we think is not worth it, I am ok